src/style.css

```diff
@@ -0,0 +1,100 @@
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --border-color: #e1e7ed;
+  --border-radius: 4px;
+  --font-family: "Avenir", serif;
+  --color: #3d4e61;
+}
+
+.searchBar {
+  position: relative;
+  margin-top: 20px;
+  height: 70px;
+}
+
+.searchBar--small {
+  height: 42px;
+}
+
+.searchBar__input {
+  display: block;
+  height: 100%;
+  width: 100%;
+  padding: 0;
+  font-family: var(--font-family);
+  font-weight: 300;
+  color: var(--color);
+  background-color: #fff;
+  border: 1px solid var(--border-color);
```

Comment:
Consider using a more descriptive variable name instead of `--border-color`. For example, `--primary-border-color` or `--input-border-color`. 🤔